warehouse/query-core/src/main/java/datawave/query/tables/ShardQueryLogic.java

```diff
@@ -914,6 +914,13 @@ protected void loadQueryParameters(ShardQueryConfiguration config, Query setting
                             + " is missing. Both are required to use a model");
         }
         
+        String noExpansion = settings.findParameter(QueryParameters.NO_EXPANSION_FIELDS).getParameterValue().trim();
+        if (org.apache.commons.lang3.StringUtils.isNotBlank(noExpansion)) {
```

Comment:
Is the specific package reference needed to avoid clashing with another StringUtils here? Not a dealbreaker, was just curious.

Reply:
The shard query logic uses the datawave version of string utils and I have a preference for the apache commons version. If you want I can use the datawave version to keep things consistent